internal/backend/dispatcher/resolve.go

```diff
@@ -32,7 +32,7 @@ func resolveEnv(ctx context.Context, svcs *wf.Services, env string) (sdktypes.En
 			return sdktypes.InvalidEnvID, err
 		}
 
-		p, err := svcs.Projects.GetByName(context.Background(), name)
```

Comment:
I am quite surprised we used Background before.

Reply:
me as well. This PR is fixing this